dart-mode.el

```diff
@@ -1202,6 +1202,10 @@ minibuffer."
 (defun dart-goto ()
   (interactive)
   (-when-let (filename (buffer-file-name))
+    (if (fboundp 'xref-push-marker-stack)
+        (xref-push-marker-stack)
```

Comment:
`dart-mode` already requires Emacs 24.5, so there's probably no reason to be paranoid about this not existing.